core/arch/arm/mm/tee_pager.c

```diff
@@ -104,6 +104,63 @@ static struct tee_pager_pmem_head tee_pager_rw_pmem_head =
 /* Number of registered physical pages, used hiding pages. */
 static size_t tee_pager_npages;
 
+#ifdef CFG_TEE_CORE_EMBED_INTERNAL_TESTS
+static struct tee_pager_stats tee_pager_stats;
```

Comment:
I would say no as everything is prefix with `tee_pager`, and it would break coherency. As it is a static variable, the following is acceptable `static struct tee_pager pager_stats pager_stats;`